### PR TITLE
Parameter Recommendations - sUSD on V3 Optimism

### PR DESCRIPTION
**Pull request by yoavigra**
Discussion - https://governance.aave.com/t/arfc-chaos-labs-parameter-recommendations-susd-on-v3-optimism-05-23-2024/17779#susd-exposure-on-aave-4

**Testing question:**
Is there any way testing on the t.sol file a parameters changes on emode category?

### CODE REVIEW

File: src/20240528_AaveV3Optimism_ChaosLabsParameterRecommendationsSUSDOnV3Optimism/ChaosLabsParameterRecommendationsSUSDOnV3Optimism.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+We recommend increasing sUSD’s non-E-Mode LTV to 60% and reducing its non-E-Mode LT to 70%. At this time, this would lead to a negligible $8 in liquidation across 4 accounts.
+Unfreeze the sUSD market on Optimism: While sUSD has repegged, we note that the ongoing deprecation of synth assets may continue to cause peg volatility. The discount rate, which dictates the value synth asset holders can receive when they redeem, will continue to be lowered in an effort to deprecate these markets. Synth asset holders may request a redemption at parity, which, in the future, may carry some time-based conditions (i.e., sUSD is sent to the redeeming user over the course of weeks or months) to reduce the potential impact on sUSD’s peg. While the Synthetix Council has an incentive to ensure redemptions of synth assets with minimal price impacts, whether to the synth assets or sUSD, these time-based conditions are not codified.
```
Comment (rustboyar) [inline src/20240528_AaveV3Optimism_ChaosLabsParameterRecommendationsSUSDOnV3Optimism/ChaosLabsParameterRecommendationsSUSDOnV3Optimism.md:38]: here i see the recommendation to unfreeze the asset, but its not implemented

File: src/20240528_AaveV3Optimism_ChaosLabsParameterRecommendationsSUSDOnV3Optimism/ChaosLabsParameterRecommendationsSUSDOnV3Optimism.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+| Parameter | Current | Rec |
+| --------- | ------- | --- |
+| LT        | 95%     | 93% |
+| LTV       | 93%     | 90% |
+
+## References
+
+- Implementation: [AaveV3Optimism](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20240528_AaveV3Optimism_ChaosLabsParameterRecommendationsSUSDOnV3Optimism/AaveV3Optimism_ChaosLabsParameterRecommendationsSUSDOnV3Optimism_20240528.sol)
+- Tests: [AaveV3Optimism](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20240528_AaveV3Optimism_ChaosLabsParameterRecommendationsSUSDOnV3Optimism/AaveV3Optimism_ChaosLabsParameterRecommendationsSUSDOnV3Optimism_20240528.t.sol)
+- [Snapshot](TODO)
```
Comment (kyzia551) [inline src/20240528_AaveV3Optimism_ChaosLabsParameterRecommendationsSUSDOnV3Optimism/ChaosLabsParameterRecommendationsSUSDOnV3Optimism.md:63]: Please remove if not applicable

**Review by kyzia551 — CHANGES_REQUESTED** [review]

File: src/20240528_AaveV3Optimism_ChaosLabsParameterRecommendationsSUSDOnV3Optimism/ChaosLabsParameterRecommendationsSUSDOnV3Optimism.md (hunk 2 above)
Comment (yoavigra) [reply src/20240528_AaveV3Optimism_ChaosLabsParameterRecommendationsSUSDOnV3Optimism/ChaosLabsParameterRecommendationsSUSDOnV3Optimism.md]: @kyzia551 Added the snapshot url

**Comment by yoavigra** [comment]
> > Discussion - https://governance.aave.com/t/arfc-chaos-labs-parameter-recommendations-susd-on-v3-optimism-05-23-2024/17779#susd-exposure-on-aave-4
> > **Testing question:** Is there any way testing on the t.sol file a parameters changes on emode category?
> 
> i think you can use: `AaveV3Optimism.POOL.getEModeCategoryData(id);`

@rustboyar Thank you, updated the test accordingly,

File: src/20240528_AaveV3Optimism_ChaosLabsParameterRecommendationsSUSDOnV3Optimism/ChaosLabsParameterRecommendationsSUSDOnV3Optimism.md (hunk 1 above)
Comment (yoavigra) [reply src/20240528_AaveV3Optimism_ChaosLabsParameterRecommendationsSUSDOnV3Optimism/ChaosLabsParameterRecommendationsSUSDOnV3Optimism.md:38]: @rustboyar Added the unfreeze of sUSD in this commit - https://github.com/bgd-labs/aave-proposals-v3/pull/346/commits/66d6d970f6ed8ddb43d44b790320ba2c8c68ff10

**Review by rustboyar — APPROVED** [review]

**Review by kyzia551 — APPROVED** [review]

**Pull request merged**